### PR TITLE
Miscellaneous fixes

### LINKED ISSUE #344: Crash Course: Update Recommending a Delegate Section
The "Recommending a Delegate" section of the crash course is out of date. It does not factor in Trainee Delegates, nor the fact that in many areas Regional Delegates handle some of these tasks not the Senior. We will also now not be suggesting new trainee delegates email all delegates with an introduction, only the Board and delegates within their region.

### PR DESCRIPTION
**Pull request by nsilvestri**
This PR serves to make a few miscellaneous corrections to blatantly out-of-date information. Specifically:

* Updating the Recommending a Delegate section to account for Trainee Delegates (fixes #344)
* Updating the Acquiring Competition Equipment section to reflect updated prices and processes (supersedes #329)
* Updating references to the old proximity policy

### CODE REVIEW

File: edudoc/delegate-crash-course/delegate_crash_course.md
Comment (dmint789) [inline edudoc/delegate-crash-course/delegate_crash_course.md:365]: I think "with a reason **for** why" would sound more natural

File: edudoc/delegate-crash-course/delegate_crash_course.md
Comment (dmint789) [inline edudoc/delegate-crash-course/delegate_crash_course.md:366]: You put 1. everywhere on accident, but the numbers should keep going up

File: edudoc/delegate-crash-course/delegate_crash_course.md
Comment (dmint789) [inline edudoc/delegate-crash-course/delegate_crash_course.md:363]: I might be missing something, but didn't you leave out some details mentioned in these steps?

File: edudoc/delegate-crash-course/delegate_crash_course.md
Comment (dmint789) [inline edudoc/delegate-crash-course/delegate_crash_course.md:363]: I'm a little confused about the wording here, because this assumes the reader is the one who wants to be a delegate, but then the steps below assume the reader is the one recommending a new delegate. This should instead say something like: "The following steps should be followed in order to appoint a new WCA Delegate:"

File: edudoc/delegate-crash-course/delegate_crash_course.md
Comment (dmint789) [inline edudoc/delegate-crash-course/delegate_crash_course.md:371]: I would change this line, I feel like it sounds a little unnatural. Maybe this? "Once all of the above steps have been completed, the new Trainee Delegate can finally begin their duties!"

**Review by dmint789 — COMMENTED** [review]
I left some comments, but good work so far!

**Review by dmint789 — CHANGES_REQUESTED** [review]
Oops, forgot to select "Request changes" to block merging :P

File: edudoc/delegate-crash-course/delegate_crash_course.md
Comment (nsilvestri) [reply edudoc/delegate-crash-course/delegate_crash_course.md:366]: This is intentional. In markdown, numbers on the rendered document will be in ascending order. You can see that it behaves correctly in the rich diff:

image

Repeatedly using `1.` is a common pattern so that if you add or remove a step, you don't have to re-number all the lines that come after it.

File: edudoc/delegate-crash-course/delegate_crash_course.md
Comment (nsilvestri) [reply edudoc/delegate-crash-course/delegate_crash_course.md:363]: I left this line unchanged from the previous version, but I agree that it can be better. I will update your suggested wording to:

> The following steps should be taken in order to appoint a new WCA Delegate:

This way the word "follow" isn't overused in one sentence.

File: edudoc/delegate-crash-course/delegate_crash_course.md
Comment (nsilvestri) [reply edudoc/delegate-crash-course/delegate_crash_course.md:371]: Will fix in the new commit.

File: edudoc/delegate-crash-course/delegate_crash_course.md
Comment (nsilvestri) [reply edudoc/delegate-crash-course/delegate_crash_course.md:365]: 👍

File: edudoc/delegate-crash-course/delegate_crash_course.md
Comment (nsilvestri) [reply edudoc/delegate-crash-course/delegate_crash_course.md:363]: This information is either irrelevant or out of date enough to be irrelevant:

* WEAT does not create a G Suite account until the delegate is promoted to Junior
* We're many years off of WCA accounts being new. Any new Trainee Delegate must be active in cubing when they are appointed, and so implicitly would already have an account with their WCA ID linked to it

The only think potentially missing that is still relevant is that a Trainee Delegate must have a photo on their account. I will add this in the fix commit.

File: edudoc/delegate-crash-course/delegate_crash_course.md
Comment (dmint789) [reply edudoc/delegate-crash-course/delegate_crash_course.md:366]: Dang, I didn't know that, thanks for letting me know!

**Review by dmint789 — APPROVED** [review]
LGTM! Anything else before I merge?

**Comment by nsilvestri** [comment]
You are good to merge. Thanks!

**Pull request merged**
